src/System.Net.Primitives/tests/FunctionalTests/System.Net.Primitives.Functional.Tests.csproj

```diff
@@ -8,27 +8,25 @@
     <OutputType>Library</OutputType>
     <UnsupportedPlatforms>FreeBSD</UnsupportedPlatforms>
   </PropertyGroup>
-  
   <!-- Help VS understand available configurations -->
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' " />
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' " />
-
   <ItemGroup>
-    <Compile Include="IPAddressMappingTest.cs" />
-    <Compile Include="IPAddressParsing.cs" />
-    <Compile Include="CookieTest\CookieContainerTest.cs" />
-    <Compile Include="CookieTest\CookiePortTest.cs" />
+    <Compile Include="Cookie.cs" />
+    <Compile Include="CookieCollection.cs" />
+    <Compile Include="CookieContainer.cs" />
+    <Compile Include="EndPoint.cs" />
```

Comment:
@stephentoub  @davidsh  @CIPop 
Yup, thanks for those tips.

I've followed the naming conventions of the System.Runtime tests (which I was just working on in another PR) - they seem to follow the opposite of the convention you have described here. Just wondering - what takes priority?